internal/serve/serve.go

```diff
@@ -214,7 +214,6 @@ func handleHTTP(o ServeOptions) *chi.Mux {
 	mux.Use(middleware.CorsMiddleware(o.CorsAllowedOrigins))
 	mux.Use(chimiddleware.RequestID)
 	mux.Use(chimiddleware.RealIP)
-	mux.Use(supporthttp.LoggingMiddleware)
```

Comment:
What are the implications of moving the logging middleware from here to line 229?